device/dell/x86_64-dell_z9100_c2538-r0/Force10-Z9100-T0/port_config.ini

```diff
@@ -0,0 +1,57 @@
+# name          lanes                 alias          index    speed
+Ethernet0       49,50           FiftyGigE1/1/1         1      50000
+Ethernet2       51,52           FiftyGigE1/1/2         1      50000
+Ethernet4       53,54           FiftyGigE1/2/1         2      50000
+Ethernet6       55,56           FiftyGigE1/2/2         2      50000
+Ethernet8       57,58           FiftyGigE1/3/1         3      50000
+Ethernet10      59,60           FiftyGigE1/3/2         3      50000
+Ethernet12      61,62           FiftyGigE1/4/1         4      50000
+Ethernet14      63,64           FiftyGigE1/4/2         4      50000
+Ethernet16      65,66           FiftyGigE1/5/1         5      50000
+Ethernet18      67,68           FiftyGigE1/5/2         5      50000
+Ethernet20      69,70           FiftyGigE1/6/1         6      50000
+Ethernet22      71,72           FiftyGigE1/6/2         6      50000
+Ethernet24      73,74,75,76     hundredGigE1/7         7      100000
```

Comment:
Just a suggestion to use one convention, either all alias name starts with lowercase or all with uppercase. Also can you check if for FiftyGig, it is `1/1/2` or `1/1/3`

Reply:
Sure will change

Reply:
I followed the convention like what was done in FTOS where it is fiftyGiGE1/1/1 and 1/1/2.

Reply:
The point is at the first letter, please use lower case as you suggested.